Fix/model statement (#154)
smartmind-team/thanosql-docs: docs/en/how-to_guides/ThanoSQL_model/SBERT.md

(query_expr)
```

__OPTIONS Clause__

```sql
OPTIONS(
    (text_col=column_name),

>     [max_epochs=VALUE],
>     [batch_size=VALUE],

Reviewer: ```suggestion
    [batch_size=VALUE],
    [max_epochs=VALUE],
```
순서 통일

    [learning_rate=VALUE],
    [overwrite={True|False}]
    )
```

The "__OPTIONS__" clause allows you to change the value of a parameter. The definition of each parameter is as follows.

> - "text_col": a column containing movie review data in the data table (str, default: 'text')

Reviewer: ```suggestion
- "text_col": the name of the column containing the text to be used for the training (str, default: 'text')
```

> - "max_epochs": number of times to train with the training dataset (int, optional, default: 1)
> - "batch_size": the size of dataset bundle utilized in a single cycle of training (int, optional, default: 16) 

Reviewer: ```suggestion
- "batch_size": the size of dataset bundle utilized in a single cycle of training (int, optional, default: 16)
- "max_epochs": number of times to train with the training dataset (int, optional, default: 1)
```
순서 통일

```sql
OPTIONS(
    (text_col=column_name),
    [table_name=expression],
    [batch_size=VALUE],

>     [result_col=column_name],

Reviewer: ```suggestion
    [result_col=column_name]
```

> )

Reviewer: ```suggestion
    )
```

```

The "__OPTIONS__" clause allows you to change the value of a parameter. The definition of each parameter is as follows.

> - "text_col": a column containing movie review data in the data table (str, default: 'text')

Reviewer: ```suggestion
- "text_col": the name of the column containing the text to be used for the vectorization (str, default: 'text')
```

- "table_name": the table name to be stored in the ThanoSQL workspace database. If a previously used table is specified, the existing table will be replaced by the new table with a 'convert_result' column (str, optional)
- "batch_size": the size of dataset bundle utilized in a single cycle of training (int, optional, default: 16)
- "result_col": defines the column name that contains the vectorized results (str, optional, default: 'convert_result')

__CONVERT Example__

An example "__CONVERT__" query can be found in [Search Text by Text](/en/tutorials/thanosql_search/search_text_by_text/).

```sql
%%thanosql
CONVERT USING movie_text_search_model
OPTIONS (
    text_col='review',
    table_name='movie_review_test',
    batch_size=32
    )
AS 
SELECT *

> FROM movie_review_test

Reviewer: `%%thanosql
CONVERT USING movie_text_search_model
OPTIONS (
    text_col='review',
    table_name='movie_review_test',
    batch_size=32,
    result_col="convert_result"
    )
AS 
SELECT *
FROM movie_review_test`

```

## __SEARCH TEXT Syntax__

Use the "__SEARCH TEXT__" statement to retrieve the desired text data.

```sql
query_statement:
    query_expr

SEARCH TEXT 
USING (model_name_expression)
OPTIONS(
    expression [ , ...]
    )
AS
(query_expr)
```
__OPTIONS Clause__

```sql
OPTIONS (
    (search_by={image|text|audio|video}),
    (search_input=expression),
    (emb_col=column_name),
    [result_col=column_name]
    )
```

The "__OPTIONS__" clause allows you to change the value of a parameter. The definition of each parameter is as follows.

- "search_by": defines the image|text|audio|video type to be used for the search (str)
- "search_input": defines the input to be used for the search (str)
- "emb_col": the column that contains the vectorized results (str)
- "result_col": defines the name of the column that contains the search results (str, optional. default: 'search_result')

__SEARCH TEXT Example__

An example "__SEARCH TEXT__" query can be found in [Search Text by Text](/en/tutorials/thanosql_search/search_text_by_text/).

```sql

> %%thanosql
> SELECT review, sentiment, score
> FROM (
>     SEARCH TEXT 
>     USING movie_text_search_model
>     OPTIONS (
>         search_by='text',
>         search_input='This movie was my favorite movie of all time',
>         emb_col='convert_result',
>         result_col='score'
>         )
>     AS 
>     SELECT * 
>     FROM movie_review_test
>     )
> ORDER BY score DESC 
> LIMIT 10
> ```

Reviewer: 해당 예시는 KEYWORD, TEXT를 둘 다 사용하므로 바꿔주는게 좋아보입니다.
```suggestion
%%thanosql
SELECT review, sentiment, score
FROM (
    SEARCH TEXT 
    USING movie_text_search_model
    OPTIONS (
        search_by='text',
        search_input='This movie was my favorite movie of all time',
        emb_col='convert_result',
        result_col='score'
        )
    AS 
    SELECT * 
    FROM movie_review_test
    )
ORDER BY score DESC 
LIMIT 10
```

## __SEARCH KEYWORD Syntax__

Use the "__SEARCH KEYWORD__" statement to retrieve the desired keyword data.

```sql
query_statement:
    query_expr

SEARCH KEYWORD 
USING (model_name_expression)
OPTIONS(
    expression [ , ...]
    )
AS
(query_expr)
```

__OPTIONS Clause__

```sql
OPTIONS (
    [lang={en|ko}],
    (text_col=column_name),
    [ngram_range=[VALUE,VALUE]],
    [top_n=VALUE],
    [diversity=VALUE],
    [use_stopwords={True|False}],
    [threshold=VALUE]
    )
```

The "__OPTIONS__" clause allows you to change the value of a parameter. The definition of each parameter is as follows.

- "lang": language to use (str, optional, 'ko'|'en' default: 'ko')

> - "text_col": a column containing movie review data in the data table (str, default: 'text')

Reviewer: ```suggestion
- "text_col": the name of the column containing the text to be used for the extract keyword (str, default: 'text')
```
